docs/management/connectors/pre-configured-connectors.asciidoc

```diff
@@ -11,6 +11,8 @@ action are predefined, including the connector name and ID.
 - Appear in all spaces because they are not saved objects.
 - Cannot be edited or deleted.
 
+NOTE: Pre-configured connectors cannot be used with cases. 
```

Comment:
To match usage in the rest of the topic.
```suggestion
NOTE: Preconfigured connectors cannot be used with cases. 
```

Reply:
Also, does "cases" refer to _any_ cases in Kibana, or just Security cases?

Reply:
Hey! Yes, this applies to all cases for all solutions.

Reply:
Thanks for confirming @cnasikas! I think this sentence is fine as is, then.